test/pkg/environment/common/expectations.go

```diff
@@ -395,10 +343,18 @@ func (env *Environment) EventuallyExpectUniqueNodeNames(selector labels.Selector
 
 func (env *Environment) eventuallyExpectScaleDown() {
 	GinkgoHelper()
+	var userNodeCount int
 	Eventually(func(g Gomega) {
-		// expect the current node count to be what it was when the test started
-		g.Expect(env.Monitor.NodeCount()).To(Equal(env.StartingNodeCount))
-	}).Should(Succeed(), fmt.Sprintf("expected scale down to %d nodes, had %d", env.StartingNodeCount, env.Monitor.NodeCount()))
+		// Get Nodes that are not part of the systempool. kubernetes.azure.com/mode=user
```

Comment:
Assuming this change it to deal with the potential of a NAP cluster getting system surge nodes created?

Reply:
yep exactly